src/components/PersonalSettings.vue

```diff
@@ -43,11 +43,11 @@
 					{{ t('integration_openai', 'Clear prompt history') }}
 				</label>
 				<button id="clear-text-prompt-history"
-					@click="clearPromptHistory(false,true)">
+					@click="clearPromptHistory(true, false)">
 					{{ t('integration_openai', 'Clear text prompts') }}
 				</button>
 				<button id="clear-image-prompt-history"
-					@click="clearPromptHistory(true,false)">
+					@click="clearPromptHistory(false, true)">
```

Comment:
original was correct

`clearPromptHistory(clearImages, clearText) {`

Reply:
Haha, yeah. I just changed the boolean order to give the correct UI behaviour.

Reply:
Fixed in: https://github.com/nextcloud/integration_openai/pull/61/commits/d68521b73536713ed5472c93fc0f66867e29bfff